plugins/modules/jenkins_node.py

```diff
@@ -230,23 +253,31 @@ def configure_node(self, present):
 
         if configured:
             if IS_PYTHON_2:
-                data = ElementTree.tostring(root)
+                data = et.tostring(root)
             else:
-                data = ElementTree.tostring(root, encoding="unicode")
+                data = et.tostring(root, encoding="unicode")
 
             self.instance.reconfig_node(self.name, data)
 
         self.result['configured'] = configured
         if configured:
             self.result['changed'] = True
 
-    def present_node(self):
+    def present_node(self, configure=True):  # type: (bool) -> bool
+        """Assert node present.
+
+        Args:
+            configure: If True, run node configuration after asserting node present.
+
+        Returns:
+            True if the node is present, False otherwise (i.e. is check mode).
+        """
         def create_node():
             try:
                 self.instance.create_node(self.name, launcher=jenkins.LAUNCHER_SSH)
             except jenkins.JenkinsException as e:
                 # Some versions of python-jenkins < 1.8.3 has an authorization bug when
```

Comment:
Maybe it's time to set that version (or some other recent one) as the minimum requirement to the module.

Reply:
1.8.3 does not exist (yet). I have proposed a patch for this issue but it hasn't been acknowledged yet.